packages/electrum/build.sh

```diff
@@ -6,8 +6,8 @@ TERMUX_PKG_VERSION="4.5.8"
 TERMUX_PKG_SRCURL=https://download.electrum.org/$TERMUX_PKG_VERSION/Electrum-$TERMUX_PKG_VERSION.tar.gz
```

Comment:
The package revision will need to be incremented.
```suggestion
TERMUX_PKG_REVISION=1
TERMUX_PKG_SRCURL=https://download.electrum.org/$TERMUX_PKG_VERSION/Electrum-$TERMUX_PKG_VERSION.tar.gz
```

Reply:
Hello, does this look good? Or do I need to change sha256 too, based on my output build?

Reply:
The SHA256 only changes when the upstream version changes.
`TERMUX_PKG_REVISION` is for downstream revisions to our package of a project.
<sup>https://github.com/termux/termux-packages/wiki/Creating-new-package#:~:text=TERMUX_PKG_REVISION</sup>

In this case, this is the 1st revision of Electrum 4.5.6 (`electrum-4.5.8-1`).
It's the upstream `electrum-4.5.8` source bundle, but we've made a revision in how we are packaging it, since you fixed the dependencies.